packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -127,16 +114,20 @@ export function InserterBlockList( {
 	}, [ filterValue, debouncedSpeak ] );
 
 	const hasItems = ! isEmpty( filteredItems );
-	const hasChildItems = childItems.length > 0;
 
 	return (
 		<div>
-			<ChildBlocks
-				rootClientId={ rootClientId }
-				items={ childItems }
-				onSelect={ onSelectItem }
-				onHover={ onHover }
-			/>
+			{ hasChildItems && (
+				<ChildBlocks rootClientId={ rootClientId }>
+					<BlockTypesList
+						// Pass along every block, as getInserterItems() will have
```

Comment:
Nitpick, I found the mention of `getInserterItems` confusing at first because it's not used at all in this file. Maybe this should mention `useBlockTypesState` instead

Reply:
👍 0cad806